system_monitor/CMakeLists.txt

```diff
@@ -211,20 +211,23 @@ install(DIRECTORY config/
 if (CATKIN_ENABLE_TESTING)
   roslint_add_test()
   find_package(rostest REQUIRED)
-  
+
   add_rostest_gtest(test_cpu_monitor
     test/test_cpu_monitor.test
     test/src/cpu_monitor/test_${CMAKE_CPU_PLATFORM}_cpu_monitor.cpp
     ${CPU_MONITOR_SOURCE}
   )
   target_link_libraries(test_cpu_monitor ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
-  add_rostest_gtest(test_hdd_monitor
-    test/test_hdd_monitor.test
-    test/src/hdd_monitor/test_hdd_monitor.cpp
-    src/hdd_monitor/hdd_monitor.cpp
-  )
-  target_link_libraries(test_hdd_monitor ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+  # Disabled since it depends on the system performing the test.
+  # TODO(icolwell): Re-enable or re-work test to focus on functionality of code rather
+  # than the system running the test.
```

Comment:
@icolwell-as can you create an issue in this repo for this TODO please?

Reply:
Issue #11